kentik_api_library/pyproject.toml

```diff
@@ -26,6 +26,20 @@ testpaths = ["tests/integration", "tests/unit"]
 [tool.pylint.master]
 fail-under=8.5
 
+[flake8]
```

Comment:
I think that the position of `[flake8]` breaks the `[tool.pylint.master]` section...

Reply:
Why You think so?

Reply:
Done